frontend/src/concepts/k8s/K8sNameDescriptionField/utils.ts

```diff
@@ -31,10 +31,25 @@ export enum LimitNameResourceType {
   /** Workbenches create routes */
   WORKBENCH,
   // TODO: Support Model Serving?
+  PVC,
 }
 /** K8s max DNS subdomain name length */
 const MAX_RESOURCE_NAME_LENGTH = 253;
 
+const MAX_PVC_NAME_LENGTH = 63;
+
+type ResourceTypeLimitsProps = {
+  [LimitNameResourceType.PROJECT]: number;
+  [LimitNameResourceType.WORKBENCH]: number;
+  [LimitNameResourceType.PVC]: number;
+};
+
+export const resourceTypeLimits: ResourceTypeLimitsProps = {
```

Comment:
```suggestion
export const resourceTypeLimits: Record<LimitNameResourceType, number> = {
```

You want a type that will be easy to maintain but also strong enough to force related changes.
<img width="1146" alt="image" src="https://github.com/user-attachments/assets/cddb485d-c4d9-4e5c-a180-3c28780d537f" />

Reply:
Ah I see, that makes sense thanks!